chamber/utils/json.py

```diff
@@ -0,0 +1,16 @@
+from django.core.serializers.json import DjangoJSONEncoder
+
+
+class ChamberJSONEncoder(DjangoJSONEncoder):
+
+    def default(self, val):
+        if isinstance(val, bytes):
+            return {
+                '__type__': bytes.__name__,
+                '__value__': [x for x in val],
+            }
+        else:
+            try:
+                return super().default(val)
+            except TypeError:
```

Comment:
Is `TypeError` the only exception that can be raised during serializing a non-serializable object?

Reply:
Good point, Django actually can throw `ValueError`.

https://github.com/python/cpython/blob/v3.8.3/Lib/json/encoder.py#L160-L180 
https://github.com/django/django/blob/2.2.14/django/core/serializers/json.py#L81-L104

Reply:
Fixed.